src/makeCalculator.js

```diff
@@ -37,7 +37,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, num) {
+      callback(this, num);
```

Comment:
You can use call method instead
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Function/call